org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/ZigBeeChannelConverterFactory.java

```diff
@@ -44,26 +44,26 @@
     /**
      * Map of all channels supported by the binding
      */
-    private final Map<String, Class<? extends ZigBeeBaseChannelConverter>> channelMap;
+    private final Map<ChannelTypeUID, Class<? extends ZigBeeBaseChannelConverter>> channelMap;
```

Comment:
Just found this while playing around with static thing types: Due to this change, the `channelMap.get(channel.getChannelTypeUID().toString())` (currently in line 149) needs to be changed to `channelMap.get(channel.getChannelTypeUID())`.